src/Headers.ts

```diff
@@ -147,4 +155,15 @@ export default class HeadersPolyfill {
       }
     }
   }
+
+  /**
+   * Returns an array containing the values
+   * of all Set-Cookie headers associated
+   * with a response
+   */
+  getSetCookie(): string[] {
+    const setCookieHeader =
+      this[NORMALIZED_HEADERS][SET_COOKIE_HEADER_KEY_NORMALIZED] ?? ''
```

Comment:
Do you think we could reuse `this.get(SET_COOKIE_HEADER_NAME)` here? That will encapsulate getting the header by its normalized name from the internal normalized headers list. It will also handle the return as `X || null` so we'd have to have an empty string fallback still.